ingestors/kafka/src/main/scala/hydra/kafka/programs/KeyAndValueSchemaV2Validator.scala

```diff
@@ -102,6 +108,24 @@ class KeyAndValueSchemaV2Validator[F[_]: Sync] private (schemaRegistry: SchemaRe
       case _ => List(Validator.valid)
     }
 
+  private def validateRequiredKeyFields(keySchema: Schema, streamType: StreamTypeV2): F[List[ValidationChain]] =
+    validateRequiredKeyFields(isKey = true, keySchema, streamType)
+
+  private def validateRequiredValueFields(valueSchema: Schema, streamType: StreamTypeV2): F[List[ValidationChain]] =
```

Comment:
Do we really need 2 separate methods for these 2 checks?

Reply:
we have the same approach in the **validateRecordRecordTypeSchemas** and it will help in case when we want to add some extra required filed(s) for key or value. On my point of view it makes code more readable.